<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="1" ignored="1" total="7" passed="4" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-11-16T20:35:32 BDT" name="Parallel Test Suite" finished-at="2024-11-16T20:35:55 BDT" duration-ms="22947">
    <groups>
      <group name="Purchase">
        <method signature="AutomationCartTest.submitOrder(java.util.HashMap)[pri:0, instance:Tests.AutomationCartTest@602e0143]" name="submitOrder" class="Tests.AutomationCartTest"/>
      </group> <!-- Purchase -->
      <group name="errorValidations">
        <method signature="ErrorValidationsTest.LoginErrorValidation()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" name="LoginErrorValidation" class="Tests.ErrorValidationsTest"/>
        <method signature="ErrorValidationsTest.ProductErrorValidation()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" name="ProductErrorValidation" class="Tests.ErrorValidationsTest"/>
      </group> <!-- errorValidations -->
    </groups>
    <test started-at="2024-11-16T20:35:32 BDT" name="Error Validations Test" finished-at="2024-11-16T20:35:51 BDT" duration-ms="18256">
      <class name="Tests.ErrorValidationsTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" started-at="2024-11-16T20:35:32 BDT" name="launchApplication" finished-at="2024-11-16T20:35:39 BDT" duration-ms="6441" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method retried="true" signature="LoginErrorValidation()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" started-at="2024-11-16T20:35:39 BDT" name="LoginErrorValidation" finished-at="2024-11-16T20:35:40 BDT" duration-ms="785" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at Tests.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" started-at="2024-11-16T20:35:40 BDT" name="tearDown" finished-at="2024-11-16T20:35:40 BDT" duration-ms="370" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" started-at="2024-11-16T20:35:40 BDT" name="launchApplication" finished-at="2024-11-16T20:35:42 BDT" duration-ms="2318" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="LoginErrorValidation()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" started-at="2024-11-16T20:35:42 BDT" name="LoginErrorValidation" finished-at="2024-11-16T20:35:43 BDT" duration-ms="907" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Incorrect email or password.] but found [Incorrect email password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.] but found [Incorrect email password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at Tests.ErrorValidationsTest.LoginErrorValidation(ErrorValidationsTest.java:23)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" started-at="2024-11-16T20:35:44 BDT" name="tearDown" finished-at="2024-11-16T20:35:45 BDT" duration-ms="617" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" started-at="2024-11-16T20:35:45 BDT" name="launchApplication" finished-at="2024-11-16T20:35:47 BDT" duration-ms="1742" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="ProductErrorValidation()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" started-at="2024-11-16T20:35:47 BDT" name="ProductErrorValidation" finished-at="2024-11-16T20:35:50 BDT" duration-ms="3600" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ProductErrorValidation -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.ErrorValidationsTest@2f1de2d6]" started-at="2024-11-16T20:35:50 BDT" name="tearDown" finished-at="2024-11-16T20:35:51 BDT" duration-ms="239" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Tests.ErrorValidationsTest -->
    </test> <!-- Error Validations Test -->
    <test started-at="2024-11-16T20:35:32 BDT" name="Submit Order Test" finished-at="2024-11-16T20:35:55 BDT" duration-ms="22947">
      <class name="Tests.AutomationCartTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Tests.AutomationCartTest@602e0143]" started-at="2024-11-16T20:35:33 BDT" name="launchApplication" finished-at="2024-11-16T20:35:39 BDT" duration-ms="5874" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:Tests.AutomationCartTest@602e0143]" started-at="2024-11-16T20:35:39 BDT" name="submitOrder" data-provider="getData" finished-at="2024-11-16T20:35:43 BDT" duration-ms="4370" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Iamking@000, product=ZARA COAT 3, email=anshika@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.AutomationCartTest@602e0143]" started-at="2024-11-16T20:35:43 BDT" name="tearDown" finished-at="2024-11-16T20:35:45 BDT" duration-ms="2153" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Tests.AutomationCartTest@602e0143]" started-at="2024-11-16T20:35:45 BDT" name="launchApplication" finished-at="2024-11-16T20:35:47 BDT" duration-ms="2414" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:Tests.AutomationCartTest@602e0143]" started-at="2024-11-16T20:35:47 BDT" name="submitOrder" data-provider="getData" finished-at="2024-11-16T20:35:51 BDT" duration-ms="4009" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Iamking@000, product=ADIDAS ORIGINAL, email=shetty@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.AutomationCartTest@602e0143]" started-at="2024-11-16T20:35:51 BDT" name="tearDown" finished-at="2024-11-16T20:35:52 BDT" duration-ms="228" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:Tests.AutomationCartTest@602e0143]" started-at="2024-11-16T20:35:52 BDT" name="launchApplication" finished-at="2024-11-16T20:35:54 BDT" duration-ms="2584" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="OrderHistoryTest()[pri:0, instance:Tests.AutomationCartTest@602e0143]" started-at="2024-11-16T20:35:54 BDT" name="OrderHistoryTest" finished-at="2024-11-16T20:35:55 BDT" depends-on-methods="Tests.AutomationCartTest.submitOrder" duration-ms="710" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderHistoryTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Tests.AutomationCartTest@602e0143]" started-at="2024-11-16T20:35:55 BDT" name="tearDown" finished-at="2024-11-16T20:35:55 BDT" duration-ms="230" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Tests.AutomationCartTest -->
    </test> <!-- Submit Order Test -->
  </suite> <!-- Parallel Test Suite -->
</testng-results>
